packages/main/src/TreeItem.js

```diff
@@ -29,6 +29,27 @@ const metadata = {
 		expanded: {
 			type: Boolean,
 		},
+		/**
+		* Defines whether the selection of tree node should be display as partially selected.
```

Comment:
minor: "should be displayed"
I guess a UA will review the texts in the end anyway?